drivers/driver.go

```diff
@@ -194,7 +194,7 @@ type LayerIDMapUpdater interface {
 
 	// SupportsShifting tells whether the driver support shifting of the UIDs/GIDs in a
 	// image and it is not required to Chown the files when running in an user namespace.
```

Comment:
```suggestion
	// image … to the provided mapping, and it is not required to Chown the files when running in an user namespace.
```
? If so, please update all such comments.